docs/config/app-configs.md

```diff
@@ -173,3 +173,30 @@ export default {
 }
 ```
 
+## head
+
+- Type: `array`
+- Default: `[]`
+
+Extra tags to inject into the page HTML `<head>`.  You can specify each tag in the form of `[tagName, { attrName: attrValue }, innerHTML?]`. For example, with a structure like this:
+
+```
+.
+├─ docs
+│  ├─ .vitepress
+│  │  └─ config.js
+|  └─ public
+|     └─ logo.png
+└─ package.json
+```
+
+You can add a custom favicon:
+
+```ts
+export default {
+  head: [
+    ['link', { rel: 'icon', href: '/public/logo.png' }]
```

Comment:
Shouldn't this be `/logo.png` only?

Reply:
Yeah it should be `/logo.png`. But also I think it would help to add maybe 1 or 2 more common meta info here. Maybe something like Facebook `og` properties?

```js
export default {
  head: [
    ['link', { rel: 'icon', href: '/logo.png' }],
    ['meta', { property: 'og:site_name', content: 'VitePress' }],
    ['meta', { property: 'og:image', content: 'https://vitepress.vuejs.org/og-default.png' }]
  ]
}
```